content/introduction.html

```diff
@@ -147,8 +147,20 @@ <h1 id="toc">Sections</h1>
     <li>12 <a href="#future">Future</a></li>
     <li>
       <p class="nested">
-        Post-graduation plans, Compensation
+        Post-graduation plans, full-time compensation
       </p>
     </li>
   </ul>
 </div>
+<div class="vbx2"></div>
+<div>
+  <h3 id="concise">Concise Version</h3>
+  <label class="switch">
+    <input id="concise-toggle" type="checkbox">
+    <span class="slider round"></span>
+  </label>
+  <p>
```

Comment:
unsure why/if it was intentional, but the preview shows the description of the toggle like this:

![image](https://user-images.githubusercontent.com/11150920/123833326-3e9bdb00-d8d4-11eb-8649-00cbac362aab.png)

maybe we can center-align the text in the description? (or move this to a sass file)
```suggestion
  <p style="text-align:center;">
```
![image](https://user-images.githubusercontent.com/11150920/123833846-c8e43f00-d8d4-11eb-8443-9d0e04f01068.png)